packages/docs/src/routes/docs/(qwikcity)/advanced/menu/index.mdx

```diff
@@ -109,12 +109,12 @@ export default component$(() => {
   return (
     <nav class="menu">
       {menu
-        ? menu.items?.map((item) => (
+        ? menu.items?.map((item, index) => (
```

Comment:
The <> should have the key as well

Reply:
You're welcome. To be honest, your lint did not give an error on the first position **<>**, so I ignored it. I added only for nested.

Reply:
I see, this could be another PR. Thanks for sharing.